src/coreclr/src/System.Private.CoreLib/src/System/Reflection/Emit/DynamicILGenerator.cs

```diff
@@ -558,7 +558,7 @@ private int GetTokenForSig(byte[] sig)
     internal class DynamicResolver : Resolver
     {
         #region Private Data Members
-        private __ExceptionInfo[] m_exceptions = null!;
+        private __ExceptionInfo[]? m_exceptions;
```

Comment:
`ilGenerator.GetExceptions()!` -> `ilGenerator.GetExceptions()` in the constructor

Reply:
Removed the `!`.